components/kyma-environment-broker/internal/process/provisioning/lms_tenant_step.go

```diff
@@ -49,8 +49,19 @@ func (s *provideLmsTenantStep) Run(operation internal.ProvisioningOperation, log
 
 	lmsTenantID, err := s.tenantProvider.ProvideLMSTenantID(pp.ErsContext.GlobalAccountID, region)
 	if err != nil {
-		errorMessage := fmt.Sprintf("Unable to request for LMS tenant ID: %s", err.Error())
-		return s.operationManager.RetryOperation(operation, errorMessage, 30*time.Second, 2*time.Minute, logger)
+		errorMessage := fmt.Sprintf("Unable to request for LMS tenant: %s", err.Error())
+		op, retry, err := s.operationManager.RetryOperation(operation, errorMessage, 30*time.Second, 3*time.Minute, logger)
```

Comment:
`RetryOperation` will mark all operation as failed after 3 minute, are you aware of that?